README.md

```diff
@@ -83,3 +112,24 @@ You should see output similar to the following:
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
+## Running the App in Docker 
+
+Within the `Dockerfile`, there are three container instances: `development`, `production` and `test`. The steps for running each container are mostly identical, other than when to notify Docker which container to build and deploy.
+
+For building the `development` container, in the terminal, run:
+
+```
+$ docker build --target dev --tag todo-app:dev . 
+```
+Note. the `development` and `dev` tags are replaced depending on which container you want to run i.e. production and test. 
+
+Once the container is built, in the terminal, run: 
+```
+$ docker run --env-file ./.env -p 5100:80 --mount type=bind,source="$(pwd)"/
+todo_app,target=/app/todo_app todo-app:dev
```

Comment:
You mount this at `/app/todo_app` inside the container, but your Dockerfile uses `/project/todo_app`. Also your dev container runs on port 5000, but this exposes port 80.
```suggestion
$ docker run --env-file .env -p 5100:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/project/todo_app todo-app:dev
```